libpod/pod.go

```diff
@@ -63,7 +61,9 @@ type PodConfig struct {
 	UsePodUTS      bool `json:"sharesUts,omitempty"`
 	UsePodCgroupNS bool `json:"sharesCgroupNS,omitempty"`
 
-	InfraContainer *InfraContainerConfig `json:"infraConfig"`
+	InfraContainer *ContainerConfig `json:"infraConfig"`
```

Comment:
I think we probably omit this - we should just look it up via `InfraContainerID` in the state, instead of storing it twice.